### PR TITLE
fix(CampaignsHistory): add the correct deliverys status

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add the correct delivery status same as Doppler app
- Add logic to show the icons

image

### CODE REVIEW

File: src/i18n/es.js [hunk 1]
```diff
@@ -27,6 +27,14 @@ const urlSiteContact = `${urlSite}/contacto/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 
 export default {
+  campaign_history: {
+    delivery_status: {
+      hardBounced: 'Rebotado Hard',
+      notOpened: 'No abierto',
```
Comment (cbernat) [inline src/i18n/es.js]:
```suggestion
      notOpened: 'No Abierto',
```

File: src/services/doppler-api-client.double.ts [hunk 2]
```diff
@@ -62,14 +62,14 @@ const campaignDeliveryCollection = {
...
-      deliveryStatus: 'opened',
+      deliveryStatus: 'notOpened',
       clicksCount: 23,
     },
     {
       campaignId: 3,
       campaignName: 'Emms 2019 preveento 1',
       campaignSubject: 'Ya comienza el dia 2. Accede a las conferencias',
-      deliveryStatus: 'opened',
+      deliveryStatus: 'softBounced',
```
Comment (cbernat) [inline src/services/doppler-api-client.double.ts]: Perhaps you could add all cases for testing right?

**Comment by cbernat** [comment]
How about adding some tests here? ❤️

File: src/services/doppler-api-client.double.ts (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/services/doppler-api-client.double.ts]: i added now

**Comment by fcoronelmakingsense** [comment]
updated
image

File: src/components/Reports/CampaignsHistory/CampaignsHistory.test.js [hunk 3]
```diff
@@ -125,4 +125,55 @@ describe('CampaignsHistory component', () => {
     // Assert
     await wait(() => expect(getByText('Manuel')).toBeInTheDocument());
   });
+
+  it('should show soft and hard bounced delivery status', async () => {
+    // Arrange
+    const campaignDeliveryCollection = {
+      items: [
+        {
```
Comment (cbernat) [inline src/components/Reports/CampaignsHistory/CampaignsHistory.test.js]: How about adding all four possibilities here?

Comment (fcoronelmakingsense) [reply src/components/Reports/CampaignsHistory/CampaignsHistory.test.js]: added

Comment (cbernat) [reply src/components/Reports/CampaignsHistory/CampaignsHistory.test.js]: Thanks @fcoronelmakingsense

**Comment by cbernat** [comment]
Veri nice, seems ready to ship! 🚀

**Pull request merged**